src/components/layout/InlineBlockList.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+
+/**
+ * @param {Object} props react props
+ * @returns {ReactElement}
+ */
+const InlineBlockList = ({ items, separator }) => {
+  const classNames = cx('fdsInlineBlockList', {
```

Comment:
I wonder if we want to give users the ability to pass in `SPACE` here (an empty string with a space), as they can sort of adjust the size of the spacing indirectly in that way.

Reply:
yeah, technically someone could do this: `<InlineBlockList separator="          "`

If we see anybody doing this, it's a sign we should probably have a second prop to control spacing:

`spacing: PropTypes.oneOf(['quarter', 'half', 'default', 'double'])`